src/Aspire.Hosting/ApplicationModel/ResourceLoggerService.cs

```diff
@@ -50,6 +58,19 @@ public void Complete(IResource resource)
             logger.Complete();
         }
     }
+
+    /// <summary>
+    /// Completes the log stream for the resource.
+    /// </summary>
+    /// <param name="name">The name of the resource.</param>
+    public void Complete(string name)
```

Comment:
Should it be an error, or at least should it be possible to find out that one tried to "complete" a log stream for a resource that does not exist?

Reply:
I'll return a bool.